commit 3e81185037cf97990e4598218f56d92dd70d6269
Author: NeilBrown <neilb@suse.de>
Date:   Tue Oct 20 13:19:10 2020 -0400

    clddb-tool was recently renamed to nfsdclddb.
    Unfortunately the nfsdcld man page wasn't told.
    
    Signed-off-by: NeilBrown <neilb@suse.de>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/nfsdcld/nfsdcld.man b/utils/nfsdcld/nfsdcld.man
index 4c2b1e80..861f1c49 100644
--- a/utils/nfsdcld/nfsdcld.man
+++ b/utils/nfsdcld/nfsdcld.man
@@ -209,12 +209,12 @@ not necessary after upgrading \fBnfsdcld\fR, however \fBnfsd\fR will not use a l
 version until restart.  A restart of \fBnfsd is necessary\fR after downgrading \fBnfsdcld\fR,
 to ensure that \fBnfsd\fR does not use an upcall version that \fBnfsdcld\fR does not support.
 Additionally, a downgrade of \fBnfsdcld\fR requires the schema of the on-disk database to
-be downgraded as well.  That can be accomplished using the \fBclddb-tool\fR(8) utility.
+be downgraded as well.  That can be accomplished using the \fBnfsdclddb\fR(8) utility.
 .SH FILES
 .TP
 .B /var/lib/nfs/nfsdcld/main.sqlite
 .SH SEE ALSO
-.BR nfsdcltrack "(8), " clddb-tool (8)
+.BR nfsdcltrack "(8), " nfsdclddb (8)
 .SH "AUTHORS"
 .IX Header "AUTHORS"
 The nfsdcld daemon was developed by Jeff Layton <jlayton@redhat.com>
